scenarioo-client/app/components/detailarea/detail-accordion/detail-accordion.component.ts

```diff
@@ -30,18 +33,49 @@ export class DetailAccordionComponent {
     @Input()
     isFirstOpen: boolean;
 
+    @Input()
+    tree: boolean;
+
+    @Input()
+    sketches: boolean;
+
+    @Input()
+    label: boolean;
+
     @Input()
     detailAccordionName: {};
 
     @Input()
     informationTree: {};
 
-    constructor() {
+    @Input()
+    relatedIssues: {};
+
+    @Input()
+    labelConfigurations: {};
+
+    selectedBranch: any;
+
+    constructor(private selectedBranchAndBuildService: SelectedBranchAndBuildService,
+                private locationService: LocationService,
+                private sketchIdsResource: SketchIdsResource) {
     }
 
     ngOnInit(): void {
+        this.selectedBranch = this.selectedBranchAndBuildService.selected();
```

Comment:
I would probably not store selectedBranch in a field, but rather access it where it is needed, since it is only needed in one of three cases.